e2e-tests/functions

```diff
@@ -4,6 +4,7 @@ exec 5>&2
 BASH_XTRACEFD="5"
 
 GIT_COMMIT=$(git rev-parse HEAD)
+BUILD_TIME=$(date --iso-8601=minutes)
```

Comment:
the command should work for both, Linux and macOS
it doesn't work for macOS :(

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/803/commits/558481f0432ba5725b2869137f09f0d9f313b394